libs/ui/packages/native/src/components/Icon/BannerIcon.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { type PropsWithChildren } from "react";
+import Svg, { Defs, LinearGradient, Rect, RectProps, Stop } from "react-native-svg";
+import styled, { useTheme } from "styled-components/native";
+
+import * as Icons from "@ledgerhq/icons-ui/native";
+import { ColorPalette } from "../../styles";
+import { Flex } from "../Layout";
+
+type IconKey = keyof typeof Icons;
+type IconPallet = { fg: string; bg: string };
```

Comment:
I think it's better to keep the same naming as we have for ColorPalette

```suggestion
type IconPallette = { fg: string; bg: string };
```